testing/tests/DevExpress.ui.widgets/scrollableParts/scrollable.scrollbar.tests.js

```diff
@@ -600,3 +601,19 @@ QUnit.test("content size should be rounded to prevent unexpected scrollbar appea
 
     assert.ok(scrollbar.$element().is(":hidden"), "scrollbar is not visible");
 });
+
+QUnit.test("scrollbar should be hidden when container size is almost similar to content size when zooming", function(assert) {
+    let scrollable = new Scrollable($('#scrollable'), {
+        'useNative': false
+    });
+
+    let verticalScroller = scrollable._strategy._scrollers['vertical'];
```

Comment:
Please use less private variables (._variable)

Reply:
Is it ok now? I minimized use of private variables. But I cannot use it at all cause I don’t see another way to specify the necessary methods in sinon.stubs.